cfn-resources/util/util.go

```diff
@@ -93,7 +93,7 @@ func NewMongoDBClient(req handler.Request, profile *string) (*mongodbatlas.Clien
 		profileInput = *profile
 	}
 
-	keys, handlerError := getAPIKeys(req, profileInput)
+	keys, handlerError := GetAPIKeys(req, profileInput)
```

Comment:
```suggestion
	keys, handlerError := getAPIKeys(req, profileInput)
```

Reply:
Reverted, thanks! I was testing something, and pushed.